### ISSUE TITLE
Navigation bar html not working when using hide corner interface

### ISSUE THREAD

**Issue by dwsalex** [jhpyle/docassemble#793]
I only want my navbar to display the interview title and some extra text. In the documentation it says:

> The interface in the upper-right corner lets the user log in, or shows a menu, or shows the exit button. This interface can be removed by setting hide corner interface to True.
> 
```
> features:
>   hide corner interface: True
```
> 
> Items you have created with [navigation bar html](https://docassemble.org/docs/initial.html#navigation%20bar%20html) will not be affected.

However when I have this feature enabled and wanna use this:

```
default screen parts:
  navigation bar html: |
    <li class="text-light">Sometext: ${somevariable}. ${"Sometext" if latest_version == publishingDate else "sometext " + latest_version}</li>
```

The text never appears in the navigation. When the corner interface is enabled it does appear just fine.(although the text by default was not white for me, which made it invisible. Possible issue?)

Possibly it's to do with the fact that `hide corner interface` removes:

`<div class="collapse navbar-collapse" id="danavbar-collapse">`

which contains the corner interface and the html generated by navigiation bar html.

Maybe I'm doing something wrong here or am not seeing something here. Thank you for your help!

**Comment by jhpyle** [comment]
Sorry, the initial behavior of `hide corner interface` did not affect the `navigation bar html`, but then in response to a user complaint I changed it to remove the `navbar-collapse` entirely, and I didn't fix the documentation.

I'll see if I can add the `navigation bar html` to a different container in the navigation bar when `hide corner interface` is in effect.

**Comment by jhpyle** [comment]
I believe this issue was fixed in 1.4.110 but if you have any further problems you can reopen this issue. Closing for now.
